src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/BookingService.kt

```diff
@@ -82,11 +83,85 @@ class BookingService(
   private val nonArrivalReasonRepository: NonArrivalReasonRepository,
   private val cancellationReasonRepository: CancellationReasonRepository,
   private val bedRepository: BedRepository,
+  private val placementRequestRepository: PlacementRequestRepository,
   @Value("\${application-url-template}") private val applicationUrlTemplate: String
 ) {
   fun updateBooking(bookingEntity: BookingEntity): BookingEntity = bookingRepository.save(bookingEntity)
   fun getBooking(id: UUID) = bookingRepository.findByIdOrNull(id)
 
+  @Transactional
+  fun createApprovedPremisesBookingFromPlacementRequest(
```

Comment:
Again, sorry for the misdirection here, but I reckon the `createApprovedPremisesBookingFromApplication` is the thing to do here